notebooks/203-meter-reader/203-meter-reader-digital.ipynb

```diff
@@ -0,0 +1,1087 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "29b161d0-8144-4b3f-88fe-deb6abd21f1c",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Digital Meter Reader \n",
+    "This notebook shows how to create a meter reader with OpenVINO Runtime. We use the pre-trained [PP-OCR](https://github.com/PaddlePaddle/PaddleOCR) to build up a inference task pipeline:\n",
+    "\n",
+    "1. Configure the screen area of the meter reader.\n",
```

Comment:
Could you use same words in workflow diagram, or add these numbers in diagram which will make the pipeline more clear.